Expand on man(1) page

vendion opened the pull request:
- [x] Have you test the code?
- [x] Have you check that the existing tests are passing?
- [x] The destination branch is **develop**?

This expands on the existing man(1) page, adding some details to the description section and adding an example section.

vendion committed:
Expand on man page for ww

Expanding on man page for www adding details to Description section as
well as adding examples.

Signed-off-by: Adam Jimerson <vendion@gmail.com>

vendion committed:
Merge branch 'add-man-page' into develop

nbari reviewed (commented):
Many thanks 👍, could you please add an example of how to create a self-signed certificate: https://go-www.com/post/how-it-works/

vendion committed:
Add example for using self-signed cert to man page

Adding an example for using a self-signed cert to the man page on top of
the example for Lets Encrypt.

Signed-off-by: Adam Jimerson <vendion@gmail.com>

vendion commented:
Sure, I was just going off the README for that which doesn't clearly state
there is a diff between "-s example.com" and "-s localhost"


On Thu, Oct 4, 2018 at 3:06 PM Nicolas Embriz <notifications@github.com>
wrote:

> *@nbari* commented on this pull request.
>
> Many thanks 👍, could you please add an example of how to create a
> self-signed certificate: https://go-www.com/post/how-it-works/
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/nbari/www/pull/8#pullrequestreview-161766051>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AAI2erRpxpaFpl04_SphdWn_jh_GnFsYks5uhlw6gaJpZM4XIsa6>
> .
>

vendion committed:
Merge branch 'add-self-signed-cert' into develop
